examples/global-boundaries.ipynb

```diff
@@ -0,0 +1,648 @@
+      ],
+      "text/plain": [
+       "<LINESTRING (-77.72 39.321, -77.756 39.334, -77.746 39.353, -77.753 39.378, ...>"
+      ]
+     },
+     "execution_count": 70,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gdf.iloc[4]['geometry'].boundary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5cb4b3e-045d-4078-b85b-e0745e1e60f5",
+   "metadata": {},
+   "source": [
+    "## Creating a GeoDataFrame of Boundaries\n",
+    "\n",
+    "Now that we have a way to get the boundary of Polygons and MultiPolygons, let's use those boundaries to create a GeoDataFrame of the geometry boundaries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "5bee50cf-1b99-4fa0-b484-d4c2d0ca6bc8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1: Calculate 'boundary' for each geometry\n",
+    "# Instead of adding it to the original gdf, directly create a new GeoSeries\n",
+    "boundaries = gdf['geometry'].apply(lambda geom: geom.boundary)\n",
```

Comment:
[`shapely.boundary`](https://shapely.readthedocs.io/en/stable/reference/shapely.boundary.html) will probably be much faster calling `.apply`, and is a better option to suggest to users. This is because `geom.boundary` operates on _one geometry at a time_, while `shapely.boundary()`  operates on the entire array at once, which is a lot faster.

Reply:
Thanks. I will swap this out as well and use `shapely.boundary` instead. 
As I was first making the example, I was getting AttributeError when running `geom.boundary` on the multipolygon geometries. So my thought process was find a way to access the `boundary` attribute on both multiply and singlepoly geom types.